Term entry sdl bigbang

cslylla opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

<!-- Please write a summary of the change, such as which topic(s) and file(s) that you have edited or created. Please also include relevant motivation and context: -->

- Added new term for General Concepts - SDLC Math Functions: Big Bang Model
- Closes #1804 

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry

### Checklist

<!-- Please check ALL the boxes: -->

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

SSwiniarski commented on content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:
```diff
@@ -0,0 +1,55 @@
+
+The Big Bang model requires no or minimal planning. Instead, the development starts with acquiring the inputs, such as:
+
+- time,
```
```suggestion
- Time.
```

SSwiniarski commented on content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:
```diff
@@ -0,0 +1,55 @@
+The Big Bang model requires no or minimal planning. Instead, the development starts with acquiring the inputs, such as:
+
+- time,
+- effort and
```
```suggestion
- Effort.
```

SSwiniarski commented on content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:
```diff
@@ -0,0 +1,55 @@
+
+- time,
+- effort and
+- resources.
```
```suggestion
- Resources.
```

SSwiniarski reviewed (changes requested):
@cslylla Strong entry, I only had a few comments. I have a question about the image. Is it your own work, or did you source it from somewhere?

cslylla commented:
@SSwiniarski Thank you, I accepted the changes. I made the image with Figma. I used icons from [Material Design](https://fonts.google.com/icons) and [Icons8](https://icons8.com/). I tried to use the same colours that Codecademy uses.

SSwiniarski reviewed (approved):
@cslylla Looks good for a second review. Complements on the image.

caupolicandiaz commented on content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:
```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Big Bang'
```
```suggestion
Title: 'Big Bang Model'
```

caupolicandiaz replied:
We have decided to updated all of the SLDC terms to include model in the title and the file name per https://github.com/Codecademy/docs/pull/2177. Please update this entry to match.

@yangc95

caupolicandiaz reviewed (changes requested):
@cslylla Looks good. Just a couple edits to match our new standards for this topic. Please tag me when you've finished making the changes. Thanks

cslylla commented:
@caupolicandiaz I updated the file and folder name to include "model" in it. I also made a minor change to the document: I changed  "model" to bold in the first sentence.

caupolicandiaz reviewed (approved):
@cslylla Looks good

SSwiniarski commented:
@cslylla Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/general/software-development-life-cycle/big-bang-model

![image](https://github.com/Codecademy/docs/assets/86081858/91f246a2-8c10-4eb4-a1bc-2f5afea2b347)
